src/App.tsx

```diff
@@ -1,9 +1,58 @@
+import { useEffect } from "react";
 import styles from "./App.module.css";
+import {faker} from "@faker-js/faker";
 
-const App = () => (
+interface Flight {
+  flightNumber: string;
+  origin: string;
+  destination: string;
+  departureTime: Date;
+  arrivalTime: Date;
+  flightDuration: number;
+}
+
+
+
+const App = () => {
+
+  const [allFlights, setFlights] = useState(0);
+  
+
+  useEffect(() => {
+
+  })
+
+
+
+  function generateFlights(): Flight{
+
+    const flight: Flight = {
+      flightNumber: faker.string.alphanumeric(6),
+      origin: faker.string.alphanumeric(3),
+      destination: faker.string.alphanumeric(3),
+      departureTime: faker.date.anytime(),
+      arrivalTime: faker.date.anytime(),
+      flightDuration: faker.number.int({min: 1, max: 8})
+    }
+
+    return flight;
+  } 
```

Comment:
Instead of return the flight, let's add the flight to the state variable storing the flights!

Here's how you could do that (with some other changes as well):
```diff
- function generateFlights(): Flight{
+ const generateFlight = (): Flight => {

    const flight: Flight = {
      flightNumber: faker.string.alphanumeric(6),
      origin: faker.string.alphanumeric(3),
      destination: faker.string.alphanumeric(3),
      departureTime: faker.date.anytime(),
      arrivalTime: faker.date.anytime(),
      flightDuration: faker.number.int({min: 1, max: 8})
    }

+ setFlights([...allFlights, flight]);
-    return flight;
  } 
```

Why the first change from function to const? In ES6 Javascript (the newest version), it is common practice to use arrow functions instead of the "function" syntax. There's nothing wrong with it, just a modern standard practice that we will use!